<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>740</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Contador de Exercícios - MediaPipe</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="camera_feed">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>800</width>
      <height>650</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(40, 40, 40);
border: 2px solid rgb(60, 60, 60);
color: rgb(150, 150, 150);</string>
    </property>
    <property name="text">
     <string>CÂMERA DESLIGADA</string>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="frame_controles">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>20</y>
      <width>341</width>
      <height>651</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(240, 240, 240);
border-radius: 8px;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QGroupBox" name="group_exercicios">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>301</width>
       <height>231</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>1. Selecione o Exercício</string>
     </property>
     <widget class="QRadioButton" name="radio_ex1">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exercício 1 (Ex: Rosca Direta)</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_ex2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>80</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exercício 2 (Ex: Agachamento)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_ex3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>120</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exercício 3 (Ex: Polichinelo)</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="radio_ex4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>160</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="text">
       <string>Exercício 4 (Ex: Abdominal)</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="group_repeticoes">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>270</y>
       <width>301</width>
       <height>91</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="title">
      <string>2. Defina as Repetições</string>
     </property>
     <widget class="QSpinBox" name="spin_repetitions">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>40</y>
        <width>261</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>11</pointsize>
        <weight>50</weight>
        <bold>false</bold>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>99</number>
      </property>
      <property name="value">
       <number>10</number>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="btn_start_camera">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>400</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>LIGAR CÂMERA</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_initial_position">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>480</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>DEFINIR POSIÇÃO INICIAL</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_start_exercise">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>560</y>
       <width>301</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>11</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(4, 128, 56);
color: white;</string>
     </property>
     <property name="text">
      <string>INICIAR EXERCÍCIO</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>